src/OpeningHours.php

```diff
@@ -150,13 +151,20 @@ public function isClosed(): bool
         return $this->isClosedAt(new DateTime());
     }
 
+    /**
+     * @TODO This should return DateTimeInterface on next major release
```

Comment:
`@TODO` should be lowercase to meet the PHPDoc spec: http://docs.phpdoc.org/references/phpdoc/tags/todo.html

Please fix both of these